Telemetry/RulesTest.cs

```diff
@@ -25,11 +67,302 @@ public void Should_Return_OK_Status()
         {
             // Act
             var request = new HttpRequest(Constants.TELEMETRY_ADDRESS + "/status");
-            request.AddHeader("X-Foo", "Bar");
+
+            var response = this.httpClient.GetAsync(request).Result;
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void CreatesRuleWithInstantCalculation()
+        {
+            // Arrange  
+            var ruleRequest = this.GetSampleRuleWithCalculation("Instant", "0");
+
+            // Act
+            var request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(ruleRequest));
+
+            var response = this.httpClient.PostAsync(request).Result;
+            var ruleResponse = JsonConvert.DeserializeObject<RuleApiModel>(response.Content);
+
+            this.disposeRulesList.Add(ruleResponse.Id); // Dispose after tests run
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal(ruleRequest.Name, ruleResponse.Name);
```

Comment:
It would nice to have equals method in RuleApiModel that can do the comparison, so you don't have write it everytime